log/test_util.go

```diff
@@ -0,0 +1,35 @@
+package log
+
+import (
+	"io"
+	"os"
+	"testing"
+
+	"github.com/klaytn/klaytn/log/term"
+	"github.com/mattn/go-colorable"
+)
+
+// Enable logging to STDERR
+// Exmaple use
+//     log.EnableLogForTest(log.LvlCrit, log.LvlTrace)
+// `normalLvl` is used in most cases
+// `verboseLvl` is used if `go test -v` flag is given
+func EnableLogForTest(normalLvl, verboseLvl Lvl) {
```

Comment:
Do you think we need both level parameters?

Reply:
- I added two levels because there was one case which I thought both are needed - https://github.com/klaytn/klaytn/blob/v1.8.4/storage/database/dynamodb_test.go#L200
- Also in general having at least `Crit` level is useful because otherwise all we'd see is 'exit 1' message.
- It does seem little unnecessary but it's just for testing, so should be no big deal.